i18n/en.json

```diff
@@ -604,6 +604,42 @@
     "translation": "Error to update the current channel."
   },
   {
+    "id": "api.command_channel_rename.name",
+    "translation": "rename"
+  },
+  {
+    "id": "api.command_channel_rename.hint",
+    "translation": "[text]"
+  },
+  {
+    "id": "api.command_channel_rename.desc",
+    "translation": "Rename the channel."
+  },
+  {
+    "id": "api.command_channel_rename.channel.app_error",
+    "translation": "Error to retrieve the current channel."
+  },
+  {
+    "id": "api.command_channel_rename.permission.app_error",
+    "translation": "You do not have the appropriate permissions to edit the channel purpose."
```

Comment:
@cpanato I think this should be `You do not have the appropriate permissions to rename the channel.`